management/attack_protection_test.go

```diff
@@ -84,16 +72,15 @@ func TestAttackProtection(t *testing.T) {
 			},
 		}
 
-		err := m.AttackProtection.UpdateSuspiciousIPThrottling(suspiciousIPThrottling)
-		if err != nil {
-			t.Error(err)
-		}
-
-		suspiciousIPThrottling, err = m.AttackProtection.GetSuspiciousIPThrottling()
-		if err != nil {
-			t.Error(err)
-		}
+		err := m.AttackProtection.UpdateSuspiciousIPThrottling(expected)
+		assert.NoError(t, err)
 
-		t.Logf("%+v\n", suspiciousIPThrottling)
+		actual, err := m.AttackProtection.GetSuspiciousIPThrottling()
+		assert.NoError(t, err)
+		assert.Equal(t, expected.GetEnabled(), actual.GetEnabled())
+		assert.Equal(t, expected.GetStage().GetPreLogin().GetRate(), actual.GetStage().GetPreLogin().GetRate())
+		assert.Equal(t, expected.GetStage().GetPreLogin().GetMaxAttempts(), actual.GetStage().GetPreLogin().GetMaxAttempts())
+		assert.Equal(t, expected.GetStage().GetPreUserRegistration().GetRate(), actual.GetStage().GetPreUserRegistration().GetRate())
+		assert.Equal(t, expected.GetStage().GetPreUserRegistration().GetMaxAttempts(), actual.GetStage().GetPreUserRegistration().GetMaxAttempts())
```

Comment:
Is there any way to unset or cleanup the values updated in all the tests on this file?

Reply:
[a000577](https://github.com/auth0/go-auth0/pull/50/commits/a0005778f2c417956d5fedcb1238da20383ef46e) Absolutely!:) Thanks!